site/src/main/mdoc/overview.md

```diff
@@ -16,117 +16,116 @@ position: 10
+You can use this `Driver` to perform operations _(`Transactions`)_ over the Database.<br>
+Those **Scala** classes, are nothing more than simple wrappers over their **Java** counterparts. But providing a more _"Scala-friendly"_ and functional API.
 
-You can create wrappers for any type `F[_]` for which you have an implementation of the `neotypes.Async` **typeclass** in scope.
+You can create wrappers for any effectual type _(`F[_]`)_ for which you have an implementation of the `neotypes.Async` **typeclass** in scope.
 The implementation for `scala.concurrent.Future` is built-in in the core module _(for other types, please read [alternative effects](alternative_effects))_.
 
 ```scala mdoc:compile-only
 import neotypes.GraphDatabase
-import org.neo4j.driver.AuthTokens
-import scala.concurrent.Future
-import scala.concurrent.ExecutionContext.Implicits.global
-
-val driver = GraphDatabase.driver[Future]("bolt://localhost:7687", AuthTokens.basic("neo4j", "****"))
-val session = driver.session
-```
-
-Please note that, you have to make sure that the driver and session are properly closed at the end of the application execution, to make sure all resources _(such as network connection)_ obtained are cleaned up properly.
-
-You can also use the `readSession` & `writeSession` helpers which will ensure that the session is properly closed after you use it.<br>
-Or, if you use other effect types instead of `Future`, for example, `IO`. Then, the creation of both the `Driver` & the `Session` will be wrapped over some kind of **Resource**.
-
-```scala mdoc:compile-only
-import neotypes.Driver
 import neotypes.generic.auto._ // Allows to automatically derive an implicit ResultMapper for case classes.
 import neotypes.implicits.syntax.string._ // Provides the query[T] extension method.
+import org.neo4j.driver.AuthTokens
 import scala.concurrent.Future
 import scala.concurrent.ExecutionContext.Implicits.global
 
 final case class Movie(title: String, released: Int)
 
-def result(driver: Driver[Future]): Future[List[Movie]] =
-  driver.readSession { session =>
-    """MATCH (movie: Movie)
-       WHERE lower(movie.title) CONTAINS "thing"
-       RETURN movie""".query[Movie].list(session)
-  }
+val driver = GraphDatabase.driver[Future]("bolt://localhost:7687", AuthTokens.basic("neo4j", "****"))
+
+val result: Future[List[Movie]] =
+  """MATCH (movie: Movie)
+     WHERE lower(movie.title) CONTAINS "thing"
+     RETURN movie""".query[Movie].list(driver)
 ```
 
+Please remember that, you have to make sure that the `Driver` is properly closed at the end of the application execution, to make sure all obtained resources _(such as network connections)_ are cleaned up properly.<br>
+**Note**: for other effect types instead of `Future`, for example `IO`. Then, the creation of the `Driver` will be wrapped over some kind of **Resource**.
```

Comment:
An optional, minor suggestion here
"for effect types other than `Future` (e.g. `IO`), the creation of the `Driver` will be managed by the effect specific implementation of **Resource**"